### DISCUSSION TITLE
Imagemagick linker error

### DISCUSSION THREAD

**Discussion by mendess** [termux/termux-packages#8241] [Q&A] [answered]
I've recently run `pkg update` and imagemagick stopped working.

The command I ran `magick path/to/wallpaper  -gravity center -extent 9:20 .wall`

But I get this error
```
CANNOT LINK EXECUTABLE "magick": cannot locate symbol "__multf3" referenced by "/data/data/com.termux/files/usr/lib/librsvg-2.so"...
```

Does anyone know anything about this?

**Comment by sk0kanik** [comment, marked as answer]
#8237
